src/allotropy/parsers/mabtech_apex/mabtech_apex_parser.py

```diff
@@ -0,0 +1,153 @@
+import pandas as pd
+
+from allotropy.allotrope.models.adm.plate_reader.benchling._2023._09.plate_reader import (
+    ContainerType,
+    DataSystemDocument,
+    DeviceSystemDocument,
+    ImageFeatureAggregateDocument,
+    ImageFeatureDocumentItem,
+    MeasurementAggregateDocument,
+    Model,
+    OpticalImagingDeviceControlAggregateDocument,
+    OpticalImagingDeviceControlDocumentItem,
+    OpticalImagingMeasurementDocumentItems,
+    PlateReaderAggregateDocument,
+    PlateReaderDocumentItem,
+    ProcessedDataAggregateDocument,
+    ProcessedDataDocumentItem,
+    SampleDocument,
+)
+from allotropy.allotrope.models.shared.definitions.custom import (
+    TQuantityValueMilliSecond,
+    TQuantityValueNumber,
+    TQuantityValueUnitless,
+)
+from allotropy.constants import ASM_CONVERTER_NAME, ASM_CONVERTER_VERSION
+from allotropy.named_file_contents import NamedFileContents
+from allotropy.parsers.mabtech_apex.mabtech_apex_contents import MabTechApexContents
+from allotropy.parsers.mabtech_apex.mabtech_apex_structure import (
+    PlateInformation,
+    Well,
+    WellList,
+)
+from allotropy.parsers.release_state import ReleaseState
+from allotropy.parsers.utils.uuids import random_uuid_str
+from allotropy.parsers.vendor_parser import VendorParser
+
+
+class MabTechApexParser(VendorParser):
+    @property
+    def display_name(self) -> str:
+        return "Mabtech Apex"
```

Comment:
Should this be "MabTech Apex" T capitalized. If not, does it make sense to change class names to to lowercase T for consistency